CONTRIBUTING.md

```diff
@@ -27,6 +27,15 @@ It's not you, it's us! _fastlane_ and its related tools receive a lot of issues
 
 One way we do this is by closing issues that we don't feel are immediately actionable. This might mean that we need more information in order to investigate. Or, it might mean that we haven't been able to reproduce it using the provided info. In this case we might close the issue while we wait for others to reproduce the problem and possibly provide some more info that unlocks the mystery.
 
+Another way we do this is by having an automated bot go through our issues and PRs. The main goal of the bot is to ensure that the issues are still relevant and reproducible. Issues can be opened, and later fall idle for a variety of reasons:
+
+* The user later decided not to use _fastlane_
+* A workaround was found, making it a low priority for the user
+* The user changed projects and/or companies
+* A new version of _fastlane_ has been released that fixed the problem
+
+No matter the reason, the _fastlane_ bot will ask for confirmation that an issue is still relevant after two months of inactivity. If the ticket becomes active again, it will remain open. If another month passes with no activity, however, the ticket will be automatically closed.
```

Comment:
My understanding is that the timing is currently
* 2 months first notification
* 1 week or 10 days later closing.